src/qt/rpcconsole.cpp

```diff
@@ -496,7 +497,7 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
     ui->detailWidget->hide();
     ui->peerHeading->setText(tr("Select a peer to view detailed information."));
 
-    consoleFontSize = settings.value(fontSizeSettingsKey, QFontInfo(QFont()).pointSize()).toInt();
+    consoleFontSize = settings.value(fontSizeSettingsKey, QFont().pointSize()).toInt();
```

Comment:
Why is this change needed? And what is the observable change?

https://doc.qt.io/qt-5/qfontinfo.html#details

Reply:
I see, but when QT_QPA_PLATFORM is set to something else, we actually want the values that are used to draw the text, not the point size of `QFont{}`, no?

Reply:
It seems there is no harm when `RPCConsole::setFontSize()` is called. With non-`qminimal` plugin Qt always could draw a correct font.

Reply:
I am not an expert in qt, and it might work for you. But in the general case it might depend on the fonts that are selected in the system settings.

Reply:
[Qt Documentation -> Drawing and Filling](https://doc.qt.io/qt-5/paintsystem.html):
> Qt will use the font with the specified attributes, or if no matching font exists, Qt will use the closest matching installed font.